lib/config/presets/internal/workarounds.ts

```diff
@@ -21,7 +21,7 @@ export const presets: Record<string, Preset> = {
       {
         matchDatasources: ['maven', 'sbt-package'],
         matchPackagePrefixes: ['commons-'],
-        allowedVersions: '!/^200\\d{5}(\\.\\d+)?/',
+        allowedVersions: '!/^200/',
       },
```

Comment:
Does this PR fix issue #14817?

Reply:
Sadly no, I just noticed that something is broken with the old preset even though the Regex is fine. I'll create another issue instead.

Reply:
See #15341